worldwind/src/commonMain/kotlin/earth/worldwind/WorldWind.kt

```diff
@@ -118,6 +118,7 @@ open class WorldWind @JvmOverloads constructor(
     private val scratchProjection = Matrix4()
     private val scratchPoint = Vec3()
     private val scratchRay = Line()
+    private var frameIndex = 0L
```

Comment:
Can you move this var below depthBits and make it protected to allow to use it in possible overriden customer classes.

Reply:
done